### PR TITLE
[EC-168] Isolate failing edge case found in property test for evm op code RETURN

### PR DESCRIPTION
**Pull request by adamsmo**
This PR changes property test to correctly check memory size.
Corner case related to EC-168 is tested in https://github.com/input-output-hk/etc-client/blob/phase/3/txExecution/src/test/scala/io/iohk/ethereum/vm/MemorySpec.scala#L161

### CODE REVIEW

File: src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala [hunk 1]
```diff
@@ -624,7 +624,12 @@ class OpCodeFunSpec extends FunSuite with OpCodeTesting with Matchers with Prope
       withStackVerification(op, stateIn, stateOut) {
         val (Seq(offset, size), _) = stateIn.stack.pop(2)
         val (data, mem1) = stateIn.memory.load(offset, size)
-        mem1.size should be >= (offset + size).toInt
+
+        if(size == UInt256.Zero){
+          mem1.size should be >= stateIn.memory.size
+        }else{
+          mem1.size should be >= (offset + size).toInt
+        }
```
Comment (rtkaczyk) [inline src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala]: Minor, style:

```
        if (size.isZero) {
          mem1.size should be >= stateIn.memory.size
        } else {
          mem1.size should be >= (offset + size).toInt
        }
```

File: src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala [hunk 2]
```diff
@@ -624,7 +624,12 @@ class OpCodeFunSpec extends FunSuite with OpCodeTesting with Matchers with Prope
       withStackVerification(op, stateIn, stateOut) {
         val (Seq(offset, size), _) = stateIn.stack.pop(2)
         val (data, mem1) = stateIn.memory.load(offset, size)
-        mem1.size should be >= (offset + size).toInt
+
+        if(size == UInt256.Zero){
+          mem1.size should be >= stateIn.memory.size
```
Comment (rtkaczyk) [inline src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala]: Not `>=`, always equal
